apps/src/StudioApp.js

```diff
@@ -1464,6 +1462,8 @@ StudioApp.prototype.displayFeedback = function (options) {
   if (this.config.level.hintPromptAttemptsThreshold !== undefined) {
     this.authoredHintsController_.considerShowingOnetimeHintPrompt();
   }
+
+  this.onFeedback(options);
```

Comment:
So it turns out `onFeedback` is no longer used anywhere _but_ integration tests (as far as Brent and I can tell).  This is good news - I was worried about side effects from this change, but it looks good.

Long term it would be nice to tear this out entirely and have tests just wrap `displayFeedback` to detect a done state.  That may be a complex change in the level tests runner though, and I'd suggest putting it in a separate PR.  In the meantime, it might be nice to rename `onFeedback` to something obviously test-specific.

Reply:
thought about renaming this method, but going to leave as-is for now (partly to keep the diff minimal, but also just to skip waiting on another test-run). I think replacing this callback with a passive test spy or something sounds good for the future though.